builtin.aqua

```diff
@@ -381,3 +382,10 @@ service Debug("debug"):
 
 service Console("run-console"):
     print(any: ⊤)
+
+-- Particle File Vault
+service Vault("vault"):
+    -- Puts data into the vault and returns virtual path
+    put(data: string) -> Path
+    -- Cats data from the vault
+    cat(path: Path) -> string
```

Comment:
Come to think of it, isn't it also possible for the file to be binary? For example, when reading from IPFS. What will happen then? I guess I should've asked in the other PR.

Reply:
That is a good point, but I don't know any use case for it. With string, you can do a lot with existent builtins, but to do smth with binary, you should have another service that can use vault.

Reply:
aqua-ipfs is reading to the vault, no? So it should be easy to do smth like
```
result <- Ipfs.get(CID)
??? <- Vault.cat(result.path)
```

Reply:
We have Ipfs.cat